software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcess.java

```diff
@@ -103,5 +105,5 @@
             "duration to wait whilst waiting for a machine to finish rebooting, and thus to become available again", Duration.minutes(30));
     
     AttributeSensor<Integer> RDP_PORT = Sensors.newIntegerSensor("rdpPort");
-    AttributeSensor<Integer> WINRM_PORT = Sensors.newIntegerSensor("winrmPort");
+    AttributeSensorAndConfigKey<Integer, Integer> WINRM_PORT = WinRmTool.PROP_PORT;
```

Comment:
Scratch that. Login port can't be configured at the entity (for both SSH and WINRM). Even if a config value is set it will be ignored. Leave it as sensor, getting the value from the location.